blog/content/post/2024-07-25-odroid-h4-getting-started.md

```diff
@@ -0,0 +1,423 @@
+---
+title: Getting started with Hardkernel ODROID H4+
+abstract: "First steps with new hardware and preparations for coreboot
+           firmware porting. A quick cookbook where to start and what to do
+           when you get your hands on a new platform and not get your hands
+           or board burnt."
+cover: /covers/r2d2.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2024-07-25
+archives: "2024"
+
+tags:
+  - Odroid
+  - coreboot
+  - firmware
+categories:
+  - Firmware
+
+---
+
+## Introduction
+
+We recently stumbled upon this interesting piece of hardware from
+[Hardkernel](https://www.hardkernel.com/), the [ODROID
+H4+](https://www.hardkernel.com/shop/odroid-h4-plus/). It is a nice, compact
+and power saving board with Alder Lake-N SoC. But that is not what makes it
+really outrageous. In the hardware and firmware world, having a public
+schematics is a rarity, almost non-existent, but this board actually [have
+one](https://wiki.odroid.com/_media/odroid-h4/hardware/adln-h4_sch_2024-0306.pdf)!
+Yes! Imagine what a firmware developer thinks seeing this... Of course,
+nothing else than open-source firmware port, obviously. In this blog post I
+would like to show you the perspective of firmware porting from scratch,
+beginning with the starting steps and the necessary preparation work you
+should do each time when dealing with new hardware.
```

Comment:
@miczyg1 maybe linking [this forum thread](https://forum.odroid.com/viewtopic.php?f=171&t=48691) for better understanding of our plans regarding OST2 and Dasharo would be good?

Also pointing to Dasharo Roadmap

Reply:
2e6545c